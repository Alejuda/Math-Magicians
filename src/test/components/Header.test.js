import { render } from '@testing-library/react';
import Header from '../../components/Header';
import { BrowserRouter as Router } from 'react-router-dom';

describe('Header', () => {
  test('Renders the header', () => {
    const { container } = render(
      <Router>
        <Header />
      </Router>
    
    );
    expect(container).toMatchSnapshot();
  });
});