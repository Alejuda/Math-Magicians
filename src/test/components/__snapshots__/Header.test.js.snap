// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Renders the header 1`] = `
<div>
  <header
    class="header"
  >
    <h1>
      MathMagicians
    </h1>
    <ul
      class="nav"
    >
      <li
        class="listitem"
      >
        <a
          activeclassname="[object Object]"
          aria-current="page"
          class="link active"
          href="/"
        >
          Home
        </a>
      </li>
      <li
        class="listitem"
      >
        <a
          activeclassname="active"
          class="link"
          href="/calculator"
        >
          Calculator
        </a>
      </li>
      <li
        class="listitem"
      >
        <a
          activeclassname="active"
          class="link"
          href="/quote"
        >
          Quote
        </a>
      </li>
    </ul>
  </header>
</div>
`;
